invenio_vocabularies/resources/serializer.py

```diff
@@ -38,13 +39,5 @@ class VocabularyL10NItemSchema(Schema):
     tags = fields.List(fields.Str(), dump_only=True)
 
 
-class VocabularyL10NListSchema(Schema):
+class VocabularyL10NListSchema(BaseListSchema):
```

Comment:
I guess you can remove that class and use directly the `BaseListSchema` class here?

Reply:
Yes, we can, just thought it would be easier to read the code when we "group" vocabulary related class, instead of calling the parent class directly. I will change it